csvGenerator.ipynb

```diff
@@ -0,0 +1,195 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "id": "6f275807",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "import csv\n",
+    "import math\n",
+    "import random\n",
+    "import requests\n",
+    "from CitesphereConnectorTest import *\n",
+    "from CitesphereConnector import CitesphereConnector"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "id": "2c0597b1",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "auth_object = authObject()\n",
+    "auth_object.authType  = 'oauth'\n",
+    "auth_object.access_token=\"access-token\"\n",
+    "connector = CitesphereConnector(\"https://diging-dev.asu.edu/citesphere-review/api\", auth_object)\n",
+    "groups=connector.get_groups()\n",
+    "#default max number of items displayed on a collection items page \n",
+    "max_size=50 "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "id": "37167759",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#download files from the collection items\n",
+    "def download_file(folder_path:str,ids:set, citesphere_token:str,):\n",
```

Comment:
there should be blanks after the commas and no comma at the end

Reply:
why do you require `ids` to be a set if you then turn it into a list?

Reply:
the file seems to downloaded multiple files, not just one, so the name is confusing